### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by matiasgarcia91**

### CODE REVIEW

File: comments/model.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    }
+  }
+)
+
+Comment.belongsTo(Cutting)
+Cutting.hasMany(Comment)
+Comment.belongsTo(User)
+User.hasMany(Comment)
+
+module.exports = Comment
```
Comment (matiasgarcia91) [inline comments/model.js]: see this icon? it's because we should leave newlines at the end of the files

File: comments/router.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+commentRouter.get(
+  '/comments',
+  async (request, response, next) => {
+    try {
+
+      const limit = request.query.limit || 12
+      const offset = request.query.offset || 0
+
+      const commentsList = await Comment.findAndCountAll({ limit, offset })
+      response.json({ commentsrows: commentsList.rows, commentstotal: commentsList.count })
```
Comment (matiasgarcia91) [inline comments/router.js]: camelCase your keys: `commentsRows, commentsTotal`

File: cuttings/router.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
+const { Router } = require('express');
+const cuttingRouter = new Router();
+const Cutting = require('./model');
```
Comment (matiasgarcia91) [inline cuttings/router.js:3]: first imports, then the rest of the code

File: events/router.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+)
+
+// Read future events
+eventRouter.get(
+  '/events',
+  async (request, response, next) => {
+    try {
+      const today = new Date();
+      const day = String(today.getDate() - 1).padStart(2, '0');
+      const month = String(today.getMonth() + 1).padStart(2, '0')
```
Comment (matiasgarcia91) [inline events/router.js:31]: check out moment.js for anything date related, great for manipulating, adding, substracting and then displaying in many ways https://momentjs.com/docs/#/displaying/

File: index.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+const eventsRouter = require('./events/router')
+const userRouter = require('./users/router')
+const cuttingRouter = require('./cuttings/router')
+const commentRouter = require('./comments/router')
+const pastEventRouter = require('./events-photo-album/router')
+const eventPhotoRouter = require('./events-photos/router')
+
+const cors = require('cors')
+const bodyParser = require('body-parser')
```
Comment (matiasgarcia91) [inline index.js]: library/modules imports first, then your files (routers, models, middlewares)

File: index.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+const app = express();
+
+const corsMiddleware = cors()
+app.use(corsMiddleware)
+
+const parserMiddleware = bodyParser.json()
```
Comment (matiasgarcia91) [inline index.js]: use express own parser `const parserMiddleware = express.json()` so you use one less dependency

**Review by matiasgarcia91 — COMMENTED** [review]
Nice Rick! looking nice and clean

File: events/router.js (hunk 4 above)
Comment (rickhekman) [reply events/router.js:31]: Great thanks!

File: cuttings/router.js (hunk 3 above)
Comment (rickhekman) [reply cuttings/router.js:3]: Great tips

File: index.js (hunk 6 above)
Comment (rickhekman) [reply index.js]: Oh yeah! That's right

**Pull request closed without merging**